packages/passwordless-auth-construct/package.json

```diff
@@ -18,7 +18,8 @@
   },
   "dependencies": {
     "@aws-amplify/auth-construct-alpha": "^0.2.0-alpha.6",
-    "@aws-sdk/client-sns": "^3.429.0"
+    "@aws-sdk/client-sns": "^3.429.0",
+    "@aws-sdk/client-ses": "^3.445.0"
```

Comment:
*Question:* Does`@aws-sdk/client-sns` need to be a higher version than `@aws-sdk/client-ses`? Should we align the two versions?

Reply:
They can be aligned. Will update 👍